tools/lib/provision.py

```diff
@@ -399,10 +399,10 @@ def main(options: argparse.Namespace) -> "NoReturn":
     run_as_root(["cp", REPO_STOPWORDS_PATH, TSEARCH_STOPWORDS_PATH])
 
     if CONTINUOUS_INTEGRATION and not options.is_build_release_tarball_only:
-        run_as_root(["service", "redis-server", "restart"])
-        run_as_root(["service", "memcached", "restart"])
-        run_as_root(["service", "rabbitmq-server", "restart"])
-        run_as_root(["service", "postgresql", "restart"])
+        run_as_root(["service", "redis-server", "start"])
+        run_as_root(["service", "memcached", "start"])
+        run_as_root(["service", "rabbitmq-server", "start"])
+        run_as_root(["service", "postgresql", "start"])
```

Comment:
Yeah, I think I agree it's reasonable to just start here; and we should know in the CI environment startup whether the services are already running.  I'd check that none of these fail if run twice; we do rerun provision and parts thereof to deal with network failures at times in CI.

Reply:
There are no errors when running these twice in CI runs. See [this run](https://github.com/priyank-p/zulip/actions/runs/151727880) based on [this commit](https://github.com/priyank-p/zulip/commit/2c5a763cdea20cfbf6022debf0d891e49f91102b).